### PR TITLE
2.0-preview

### PR DESCRIPTION
**Pull request by cdata**
Port to 2.0

### CODE REVIEW

File: test/index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
 <body>
   <script>
     WCT.loadSuites([
-      'paper-button.html',
+      'paper-button.html?wc-shadydom=true&wc-register=true',
```
Comment (e111077) [inline test/index.html]: wc-register -> wc-ce

**Comment by e111077** [comment]
I'm surprised how few changes were required for this element.

**Comment by e111077** [comment]
please rebase

**Comment by notwaldorf** [comment]
@e111077 donzo

**Comment by e111077** [comment]
@notwaldorf needs moar gif

**Pull request merged**